web/client/components/data/featuregrid/toolbars/Toolbar.jsx

```diff
@@ -115,4 +115,11 @@ module.exports = ({events = {}, mode = "VIEW", selectedCount, hasChanges, hasGeo
             visible={mode === "VIEW"}
             onClick={events.sync}
             glyph="map-filter"/>
+        <TButton
```

Comment:
This button should be present visibile only if the widget builder is enabled. You should also be able to disable it via configuration.